spec.html

```diff
@@ -23527,7 +23527,8 @@
     <!-- es6num="19.3.3" -->
     <emu-clause id="sec-properties-of-the-boolean-prototype-object">
       <h1>Properties of the Boolean Prototype Object</h1>
-      <p>The Boolean prototype object is the intrinsic object %BooleanPrototype%. The Boolean prototype object is an ordinary object. It is not a Boolean instance and does not have a [[BooleanData]] internal slot.</p>
```

Comment:
Nit on @rwaldron's proposed verbiage: it is a run on sentence, and should either be split into two sentences or use a semicolon instead of a comma.